Add key for menu option name (category)

ryanmitchell opened the pull request:
Naming is hard :)

If you're happy with this approach I'll apply it to the emails

And then printer...
And then kitchen display...

sampoyigi commented:
Yea database naming is messy, especially the menu_option menu_option_value ones.

Maybe `order_option_group`? And you can use `order_menu_option_id` on `order_menu_options` to query `menu_options` directly.

ryanmitchell commented:
Updated to order_option_group and added to email. Is there anywhere else I need to cover the change to the output?

order_menu_option_id references the id of `menu_item_options`, not `menu_options`. I would love it if there was an easier way as I use this in printer and KD too, so if you’ve got one please show me :)



> On 15 Apr 2021, at 18:08, Sam Poyigi ***@***.***> wrote:
> 
> 
> Yea database naming is messy, especially the menu_option menu_option_value ones.
> 
> Maybe order_option_group? And you can use order_menu_option_id on order_menu_options to query menu_options directly.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/tastyigniter/TastyIgniter/pull/733#issuecomment-820590238>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AAAMVO7NKJTII7OEIITHUO3TI4MPNANCNFSM4255XNPQ>.
>

BreakSecurity commented:
What is the purpose of this?

sampoyigi commented:
> order_menu_option_id references the id of `menu_item_options`, not `menu_options`. I would love it if there was an easier way as I use this in printer and KD too, so if you’ve got one please show me :)

My bad.. I've updated the code to use Menu_item_options instead.

I think what's left is the changes on the order invoice output and the order confirmation page.

ryanmitchell commented:
Looks good. I'll try and work on those over the weekend unless someone beats me to it.
